src/DHI.Mesh.Test/DHI.Mesh.Test.csproj

```diff
@@ -10,7 +10,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.11.0" />
     <PackageReference Include="DHI.DFS" Version="20.0.0" />
-    <PackageReference Include="NetTopologySuite.IO.GeoJSON" Version="1.15.0" />
+    <PackageReference Include="NetTopologySuite.IO.GeoJSON" Version="2.0.4" />

```

Comment:
This is 2.0.4 and not 2.4. Does these not follow each other?

Reply:
No, they don't follow each other. Also, this is only used in the test-project anyway.